settings/rest-api.php

```diff
@@ -161,7 +161,7 @@ function rest_update_provider_status( WP_REST_Request $request ) {
 			break;
 
 		case 'disable':
-			$result = new WP_Error( 'todo_pending_194', 'TODO pending #194.', array( 'status' => 501 ) );
+			$result = Two_Factor_Core::disable_provider_for_user( $user_id, $provider );
```

Comment:
This method doesn't exist yet, it'll require changes to [Two_Factor_Core](https://github.com/WordPress/two-factor/blob/master/class-two-factor-core.php#L1796)

Reply:
Do we plan to hold off the merge of this PR until the method is added to [Two_Factor_Core](https://github.com/WordPress/two-factor/blob/master/class-two-factor-core.php#L1796)?

Reply:
This is all still flagged for development only, so I think we can merge the UI beforehand.